app/containers/ItunesProvider/DetailContainer/index.js

```diff
@@ -0,0 +1,160 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { get } from 'lodash';
+import { Card, Skeleton } from 'antd';
+import styled from 'styled-components';
+import { injectIntl } from 'react-intl';
+import { injectSaga } from 'redux-injectors';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { selectSongContainer, selectSongsData, selectSongsError, selectLoading } from '../selectors';
+import { songContainerCreators } from '../reducer';
+import { songContainerSaga } from '../saga';
+import T from '@components/T';
+import LazyImage from '@app/components/LazyImage';
+import AudioPlayer from '@app/components/AudioPlayer';
+
+const { Meta } = Card;
+
+const LinkButton = styled.a`
+  && {
+    margin-top: 1em;
+    padding: 14px 25px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
font-family shouldn't ideally change in the app. we set it up in the app/global-styles.js

Reply:
got it